product_secondary_unit/views/product_views.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">Product template Secondary Unit</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="groups_id" eval="[(4, ref('product.group_uom'))]"/>
```

Comment:
In v12.0 the security group is defined in uom module, see below:
[https://github.com/odoo/odoo/blob/f3f84306004437b6c99b1698e4d9709db3220cc7/addons/uom/security/uom_security.xml#L4](url)

Reply:
Thank you, fixed and finished migrate now.